common/test/acceptance/pages/lms/instructor_dashboard.py

```diff
@@ -45,6 +45,15 @@ def select_cohort_management(self):
         cohort_management_section.wait_for_page()
         return cohort_management_section
 
+    def select_discussion_management(self):
+        """
+        Selects the Discussion tab and returns the DiscussionmanagementSection
+        """
+        self.q(css='[data-section="discussions_management"').first.click()
+        discussion_management_section = DiscussionManagementSection(self.browser)
+        discussion_management_section.wait_for_page()
```

Comment:
We may need a "wait_for_ajax" if we end up doing an ajax call to get some page initial state and experience flaky test failures. However, I wouldn't add it prematurely... just something to consider given the comment above in select_cohort_management.